src/rdflib_endpoint/__main__.py

```diff
@@ -4,9 +4,10 @@
 
 import click
 import uvicorn
-from rdflib import Dataset
+from rdflib import Dataset, Graph
 
 from rdflib_endpoint import SparqlEndpoint
+from pycottas import COTTASStore
```

Comment:
@arenas-guerrero-julian  We need to do this import only when we try to load a `.cottas` file, otherwise this statement will be triggered all the time (even for people using the CLI for regular RDF files), and the CLI will fail for anyone not working with cottas and that have not installed `rdflib-endpoint[cli,cottas]` with the optional pycottas dependency